cmd/root.go

```diff
@@ -1,51 +1,31 @@
 package cmd
 
 import (
-	"github.com/spf13/cobra"
-	"fmt"
+	"log"
 	"os"
+
+	"github.com/spf13/cobra"
 )
 
 const NodepingTokenKey = "NODEPING_TOKEN"
+
 var nodepingToken string
 
 var rootCmd = &cobra.Command{
```

Comment:
Would be nice to move these out of global scope.

Reply:
True, I was thinking the same, but didn't want to put "too much" cleanup in. I'm also not sure off the top of my head how to do this easily. (I have some untested ideas.)